src/nvim/os/wstream.c

```diff
@@ -240,7 +240,7 @@ static void write_cb(uv_write_t *req, int status)
   kmp_free(WRequestPool, wrequest_pool, data);
 }
 
-static void release_wbuffer(WBuffer *buffer)
+void release_wbuffer(WBuffer *buffer)
```

Comment:
I'm not sure what to think about this change.  On the one hand, it seems like WBuffer is an implementation detail and not really meant to be exposed outside of wstream.c (at least manipulating it in this way).  On the other hand, you need to free the object.  Does the name need to change to `wbuffer_release`?